client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -20,7 +22,7 @@ const ToolBar = ({ onZoomChange, currentPage, onPageChange, totalPages })=>{
 	}, [currentPage]);
 
 	const handleZoomButton = (delta)=>{
```

Comment:
So when I proposed naming this `handleZoomIncrement` before, I assumed this would be just for the +/- buttons. Now it is being used also for the "fit" buttons, which means `calculateZoom` has to awkwardly calculate a delta instead of being able to more straightforwardly just set the zoom level it calculated, so I propose one of two refactors:

1) Have `calculateZoom` just `setZoomLevel(desiredZoom - margin)` at the end, and rename this to `handleZoomIncrement` to distinguish it from the `fit` buttons.
2) Rename this back to `handleZoomChange`, but have the input just be the zoom level you want to set instead of a delta, then update the +/- buttons so they send `zoomLevel +- 20`.

2 might be better so you can still have a centralized place to perform rounding and clamping.